torch/onnx/symbolic_opset11.py

```diff
@@ -273,7 +273,9 @@ def masked_scatter(g, self, mask, source):
 
 
 def _len(g, self):
-    return g.op("SequenceLength", self)
+    if self.type().isSubtypeOf(torch._C.ListType.ofTensors()):
```

Comment:
Looks good. Do we have tests to cover both branches in this symbolic?

Reply:
yes, test_len covers the if branch, test_len_list covers the else branch